src/main/scala/com/evolutiongaming/cluster/ActorRefExtension.scala

```diff
@@ -0,0 +1,5 @@
+package com.evolutiongaming.cluster
+
+import akka.actor.{ActorRef, Extension}
+
+class ActorRefExtension(val ref: ActorRef) extends Extension
```

Comment:
Than you need to implement ExtensionId and pass `props` as argument and start actor in `ExtensionId.createExtension`